### PR TITLE
Ful 16279

### PR DESCRIPTION
**Pull request by giulianfazio**

### CODE REVIEW

File: beekeeper-dependency-updates-plugin/src/test/groovy/io/beekeeper/gradle/dependencies/DependencyUpdatesCheckPluginTest.groovy [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        workspace.buildFile << """
+        plugins {
+            id 'java'
+            id '${DependencyUpdatesCheckPlugin.IDENTIFIER}'
+        }
+        """
+    }
+
+    def "it should apply the spotbugs plugin"() {
```
Comment (lguzik-bkp) [inline beekeeper-dependency-updates-plugin/src/test/groovy/io/beekeeper/gradle/dependencies/DependencyUpdatesCheckPluginTest.groovy]:
```suggestion
    def "it should apply the versions plugin"() {
```

File: beekeeper-dependency-updates-plugin/build.gradle [hunk 2]
```diff
@@ -0,0 +1,31 @@
+dependencies {
+    implementation "com.github.ben-manes:gradle-versions-plugin:0.27.0"
+}
+
+gradlePlugin {
+    plugins {
+        dependencyUpdatesPlugin {
+            id = 'io.beekeeper.gradle.plugins.dependency-updates'
+            implementationClass = 'io.beekeeper.gradle.dependency.DependencyUpdatesCheckPlugin'
```
Comment (lguzik-bkp) [inline beekeeper-dependency-updates-plugin/build.gradle]:
```suggestion
            implementationClass = 'io.beekeeper.gradle.dependencies.DependencyUpdatesCheckPlugin'
```

**Comment by giulianfazio** [comment]
@lguzik-bkp thank you for catching up the remaining typos!! :D

**Comment by lguzik-bkp** [comment]
And even tests are working now >:D

File: beekeeper-code-analysis-plugin/src/test/groovy/io/beekeeper/formatter/CodeAnalysisPluginTest.groovy [hunk 3]
```diff
@@ -29,7 +28,7 @@ class CodeAnalysisPluginTest extends Specification {
         """
     }
 
-    def "it should apply the spotbugs plugin"() {
+    def "it should apply the version plugin"() {
```
Comment (lguzik-bkp) [inline beekeeper-code-analysis-plugin/src/test/groovy/io/beekeeper/formatter/CodeAnalysisPluginTest.groovy]: You changed it in the wrong place ;)

Comment (giulianfazio) [reply beekeeper-code-analysis-plugin/src/test/groovy/io/beekeeper/formatter/CodeAnalysisPluginTest.groovy]: lol. I'm amending this cause it's a very tiny change

**Pull request merged**